test/entrypoint.sh

```diff
@@ -51,7 +51,9 @@ if [ -n "${PKCS11_PROXY_SOCKET:-}" ]; then
 fi
 
 if [[ $# -eq 0 ]]; then
-    exec ./start.py
+    PS1=foo
```

Comment:
Can you add a small comment explaining why the `PS1` modification is required here? I have a vague memory of you explaining this to me in another context but I can't remember the reason now and without more detail it looks like an accidental diff.

Reply:
I think maybe a different version of this had `set +x` and a comment, which is what I intended to incorporate. entrypoint.sh uses `set -u` (error on unset variables) at the top. Since it sources the `activate` script that setting applies to that script too, and `activate` checks the value of `$PS1` (to update your bash prompt). Will fix.